calcom/cal.com
# adds multiline/markdown bio

## What does this PR do?

Adds multiline support for user bio. Also allows limited Markdown support like event type descriptions. Also turns user bio text field into a textarea, and makes it autosize itself.

Fixes #6204

<img width="734" alt="image" src="https://user-images.githubusercontent.com/26844/210021781-2cbbe894-18cf-40d2-b981-86a32ad3b563.png">

<img width="727" alt="image" src="https://user-images.githubusercontent.com/26844/210021790-90e8bdd7-2be6-4fcc-a41e-f3758c86c1ba.png">


**Environment**: Staging(main branch)

## Type of change


## How should this be tested?

* Create a multiline/markdown bio. Notice that the textarea expands.
* Go to profile page and notice markdown/multiline bio

## Checklist



- I haven't added tests that prove my fix is effective or that my feature works
